Create slides-trusted-networks.md (#36)
ietf-wg-add/draft-add-requirements: slides-trusted-networks.md

#DNS configuration on explicitly trusted networks

> * Client can authenticate the identity of the network (or pre-existing relationship with the network)

Reviewer: ```suggestion
* Client can authenticate the identity of the network (or pre-existing relationship with the network) and has authorized it to override local DNS settings
```

Author: I think you mean user has authorized the client to override local DNS setting.

Reviewer: OK, "user" might be better than "client".  I think the user is really authorizing the "network", though.  This authorization is specific to one network, not a general authorization for the "client" when it connects to any future network.

Author: Agreed. I will update the above line as follows:
Client can authenticate the identity of the network (or pre-existing relationship with the network) and the user has authorized the client to override local DNS settings for a specific network.

> * BYOD devices joining Enterprise network without any MDM and configuration profile (e.g., using EAP-pwd, EAP-PSK).
> * IoT devices joining Enterprise network without a device management tool

Reviewer: ```suggestion
    - BYOD devices joining Enterprise network without any MDM and configuration profile (e.g., using EAP-pwd, EAP-PSK).
    - IoT devices joining Enterprise network without a device management tool
```

#Goals

* Standardized discovery mechanism for BYOD and IoT devices.
* Discover local names to use Enterprise DNS server (similar to split DNS configuration in IKEv2)

>    - Discover if the Enterprise network allows split DNS configuration

Reviewer: ```suggestion
   - Discover if the Enterprise network offers a split DNS configuration
```
